src/server/pps/cmds/cmds.go

```diff
@@ -438,6 +439,32 @@ each datum.`,
 		follow      bool
 		tail        int64
 	)
+
+	// prettyLogsPrinter helps to print the logs recieved in different colours
+	prettyLogsPrinter := func(message string) {
+		informationArray := strings.Split(message, " ")
+		if len(informationArray) > 1 {
+			debugString := informationArray[1]
+			debugLevel := strings.ToLower(debugString)
+			var debugLevelColoredString string
+			if debugLevel == "info" {
+				debugLevelColoredString = color.New(color.FgGreen).SprintFunc()(debugString)
+			} else if debugLevel == "warning" {
+				debugLevelColoredString = color.New(color.FgYellow).SprintFunc()(debugString)
+			} else if debugLevel == "error" {
+				debugLevelColoredString = color.New(color.FgRed).SprintFunc()(debugString)
```

Comment:
You could just use `Sprint` here instead of `SprintFunc`

Reply:
I follow `SprintFunc` because it was widely used in the codebase, wanted to be consistent :sweat_smile: , made the changes though

Reply:
I follow `SprintFunc` because it was widely used in the codebase, wanted to be consistent :sweat_smile: , made the changes though